labwork5/Regular expressions.py

```diff
@@ -0,0 +1,32 @@
+import re
+
+# task 1.1
+text = 'gswdhc@epa.com djfbgh fdfivgidebvfu@mail.ru vbdsbvj sonskvn@gmail.com'
+print(re.findall(r'\w+@\w+\.\w+', text))
+
+
+# task 1.2
+text = 'wreyru erfh elfjrlfje ldfl nfn fn'
+all_words = re.findall(r'\w+', text)
+for word in all_words:
+    if len(word) in (3, 4, 5):
```

Comment:
you can use {3,5} instruction instead in your regular expression. It's just a tip.